### ISSUE TITLE
Update UI to point back to a linked Artifact by content-ref field in File objects

### ISSUE THREAD

**Issue by YungBinary** [LuatixHQ/opencti#1845]
## Use case

As pointed out by @nor3th [here](https://github.com/OpenCTI-Platform/connectors/pull/601#discussion_r795960645), the `content_ref` field is available for Connector Developers to update within StixFile objects. The following is an example of how we want to update this field:

```python
    response = self.helper.api.stix_cyber_observable.upload_artifact(**kwargs)
    new_artifact_standard_id = response["standard_id"]
    self.helper.api.stix_cyber_observable.update_field(
        id=stix_file_observable_id,
        input={"key": "content_ref", "value": new_artifact_standard_id},
    )
```

## Current Workaround

Create a Relationship as a `related-to` to the newly uploaded Artifact.

## Proposed Solution

Show a click-able reference that points to the Artifact object within `Details` while viewing the File (StixFile).

**Comment by nor3th** [comment]
Hey @YungBinary 

This won't work, because we decided to implement nested relationships (like `content_ref`) as a relationship instead of as an attribute containing a list of references. And this relationship (like a few others) is still missing.

The mentioned relationship is part of this PR (https://github.com/OpenCTI-Platform/opencti/pull/1803/files#diff-8f804f0d600cb54e6d42acc99961742916a240a86e8281cac53e41ccb38e16c5R893), I only have to turn the relationship around first (since I only recently noticed the direction issue with nested relationships). Here more details on it: https://github.com/OpenCTI-Platform/opencti/issues/1805#issuecomment-1013946606

After its implementation, you'll be able to simply do
```
api_client.stix_cyber_observable_relationship.create(
    fromId=artifact["id"],
    toId=stix_file["id"],
    relationship_type="content",
)
```

Edit: Also once the PR is accepted, you'll be able to use this relationship in the UI interface as well.

Regards
